packages/client-react-streaming-support/src/DataTransportAbstraction/useTransportValue.tsx

```diff
@@ -1,5 +1,5 @@
 "use client";
-import { useContext, useEffect, useState } from "rehackt";
+import { useContext, useEffect, useState } from "react";
```

Comment:
With all the exports conditions we have in play for these packages now, we don't need `rehackt` anymore.